RELEASE.md

```diff
@@ -1,5 +1,8 @@
 # odjango
 
+## next
+Add the folders inside rest_framework_app
```

Comment:
must indicate if it is a patch, a minor or a major :
* p: rest_framework_app folder was added in manifest to fix package build